src/components/DateTime/splDatetimeToIso.js

```diff
@@ -0,0 +1,46 @@
+export function splDatetimeToIso(datetime) {
+	// Check if datetime is a number or BigInt
+	if ((typeof datetime !== 'bigint') && (typeof datetime !== 'number')) {
+		return 'Invalid Date';
+	}
+	const BigIntDatetime = BigInt(datetime); // Convert to BigInt for correct bitwise operations
```

Comment:
This should be rather "assert" - we don't want to transport splang datetime in anything else than BigInt - so this should fail (at least in the testing setup) to indicate that problematic scenario.